common/bootstrap/templates/block-interval-notifier.yaml

```diff
@@ -0,0 +1,33 @@
+{{ if .Values.blockIntervalNotifier.enabled }}
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  namespace: argocd
```

Comment:
if you wanna use secrets, the resource should be in the same namespace as the secret it uses

Reply:
thanks you